### ISSUE TITLE
For the futur, I ask that the size of the fonts be proportional to the size of the widgets.  

### ISSUE THREAD

**Issue by philippe-warnotte** [FrSkyRC/ETHOS-Feedback-Community#4065]
For a good view of telemetry and timers.

**Comment by MikeActron** [comment]
Yes, the font should really be larger or adjustable in size. The X20 series in particular has such a high-resolution LCD and then such small fonts are used. Other manufacturers can do the same.

**Comment by philippe-warnotte** [comment]
![Powerbox](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/165813792/c0504b0f-5af8-4a1d-bb2d-0e2e0ba9a4db)
Exemple

**Comment by philippe-warnotte** [comment]
Dommage, les timers chez Powerbox sont bien plus grands.

![image](https://github.com/user-attachments/assets/c1dd32e2-69b0-4cea-93e5-119a8b75ec3a)
